modules/apps/change-tracking/change-tracking-rest-test/src/testIntegration/java/com/liferay/change/tracking/rest/resource/v1_0/test/BaseCTCollectionResourceTestCase.java

```diff
@@ -2026,4 +2033,6 @@ public String toString() {
 	private com.liferay.change.tracking.rest.resource.v1_0.CTCollectionResource
 		_ctCollectionResource;
 
+	private String _originalName;
+
```

Comment:
setUp and tearDown run for every test cases. If this only happens with the new test case, let's set the PrincipalThreadLocal locally in a try-finally block.